### ISSUE TITLE
`crictl --version` returns 1.11.0 for v1.11.1.

### ISSUE THREAD

**Issue by Random-Liu** [kubernetes-sigs/cri-tools#353]
Because the cherrypick of https://github.com/kubernetes-incubator/cri-tools/pull/339 was missed in v1.11.1, the version number is not updated properly.

We've cherrypicked https://github.com/kubernetes-incubator/cri-tools/pull/339 to v1.11.2 to fix this problem in the future, but it still worth filing this issue, so that users who hit this problem can find a reason here.

**Comment by Random-Liu** [comment]
I'll close this issue when 1.11.2 is cut.

**Comment by ktzee** [comment]
It currently only returns "unknown".
https://github.com/kubernetes-sigs/cri-tools/blob/f0cc060d7fdffd9a36892cb7e5c4b1301a30881e/pkg/version/version.go

`$ rpm -qi cri-tools
Name        : cri-tools
Version     : 1.11.0
Release     : 2.dev.git19b7255.fc29

$ which crictl
/usr/bin/crictl

$ crictl -v
crictl version unknown
`

The same also happens with the package on Fedora repositories.

**Comment by feiskyer** [comment]
Has been fixed in [v1.11.1](https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.11.1).

```sh
./crictl --version
crictl version 1.11.1

 ./crictl -v
crictl version 1.11.1
```
